i18n/README.vi-vn.md

```diff
@@ -0,0 +1,119 @@
+| `postgrest-{lang}`    | [`JS`](https://github.com/supabase/postgrest-js) | [`C#`](https://github.com/supabase/postgrest-csharp) \| [`Dart`](https://github.com/supabase/postgrest-dart) \| [`Python`](https://github.com/supabase/postgrest-py) \| [`Rust`](https://github.com/supabase/postgrest-rs) \| [`Ruby`](https://github.com/supabase/postgrest-rb) \| [`Go`](https://github.com/supabase/postgrest-go) |
+| `realtime-{lang}`     | [`JS`](https://github.com/supabase/realtime-js)  | [`C#`](https://github.com/supabase/realtime-csharp) \| [`Dart`](https://github.com/supabase/realtime-dart) \| [`Python`](https://github.com/supabase/realtime-py) \| `Rust` \| `Ruby` \| `Go`                                                                                                                                        |
+| `gotrue-{lang}`       | [`JS`](https://github.com/supabase/gotrue-js)    | [`C#`](https://github.com/supabase/gotrue-csharp) \| [`Dart`](https://github.com/supabase/gotrue-dart) \| [`Python`](https://github.com/supabase/gotrue-py) \| `Rust` \| `Ruby` \| `Go`                                                                                                                                              |
+
+<!--- Remove this list if you're traslating to another language, it's hard to keep updated across multiple files-->
+<!--- Keep only the link to the list of translation files-->
+
+## Các bản dịch
+
+- [Arabic | العربية](/i18n/README.ar.md)
+- [Albanian / Shqip](/i18n/README.sq.md)
+- [Danish / Dansk](/i18n/README.da.md)
+- [Dutch / Nederlands](/i18n/README.nl.md)
+- [English](https://github.com/supabase/supabase)
+- [French / Français](/i18n/README.fr.md)
+- [German / Deutsch](/i18n/README.de.md)
+- [Greek / Ελληνικά](/i18n/README.gr.md)
+- [Hebrew / עברית](/i18n/README.he.md)
+- [Hindi / हिंदी](/i18n/README.hi.md)
+- [Nepali / नेपाली](/i18n/README.ne.md)
+- [Indonesian / Bahasa Indonesia](/i18n/README.id.md)
+- [Italiano / Italian](/i18n/README.it.md)
+- [Japanese / 日本語](/i18n/README.jp.md)
+- [Korean / 한국어](/i18n/README.ko.md)
+- [Norwegian (Bokmål) / Norsk (Bokmål)](/i18n/README.nb-no.md)
+- [Persian / فارسی](/i18n/README.fa.md)
+- [Polish / Polski](/i18n/README.pl.md)
+- [Portuguese / Portuguese](/i18n/README.pt.md)
+- [Portuguese (Brazilian) / Português Brasileiro](/i18n/README.pt-br.md)
+- [Romanian / Română](/i18n/README.ro.md)
+- [Russian / Pусский](/i18n/README.ru.md)
+- [Sinhala / සිංහල](/i18n/README.si.md)
+- [Spanish / Español](/i18n/README.es.md)
+- [Simplified Chinese / 简体中文](/i18n/README.zh-cn.md)
+- [Swedish / Svenska](/i18n/README.sv.md)
+- [Thai / ไทย](/i18n/README.th.md)
+- [Traditional Chinese / 繁体中文](/i18n/README.zh-tw.md)
+- [Turkish / Türkçe](/i18n/README.tr.md)
+- [Ukrainian / Українська](/i18n/README.uk.md)
+- [Vietnamese / Tiếng Việt](/i18n/README.vi-VN.md)
+- [Danh sách các bản dịch](/i18n/languages.md) <!--- Keep only this -->
```

Comment:
The rest of languages should be removed, read the HTML comment above: 
> *Remove this list if you're traslating to another language, it's hard to keep updated across multiple files*

Reply:
Resolved. Thanks for the heads-up!